checkers/commentFormatting_checker.go

```diff
@@ -74,6 +74,6 @@ func (c *commentFormattingChecker) specialChar(r rune) bool {
 	}
 }
 
-func (c *commentFormattingChecker) warn(cg *ast.CommentGroup) {
-	c.ctx.Warn(cg, "put a space between `//` and comment text")
+func (c *commentFormattingChecker) warn(comment *ast.Comment) {
```

Comment:
why so? :)

Reply:
Looks like a bug made before, but not sure.

Reply:
Previously, it would complain on the beginning of the comment block.
Now it should report the exact comment line instead.